### PR TITLE
Turns

### PR DESCRIPTION
**Pull request by adamRoar**
Implemented logic for turns to alternate between RED and BLACK players after each move. Further added cases where turn does not switch if a player can make a double or triple jump.

### CODE REVIEW

File: model/model.py [hunk 1]
```diff
@@ -131,3 +162,13 @@ def check_move_type(self, start, end, piece_at_start, direction):
                 if jumped_piece is not None and jumped_piece.color != piece_at_start.color:
                     move_type = MoveType.JUMP
         return move_type
+
+    def can_jump(self, tile: Tile) -> bool:
+        jump_directions = [Tile(-2, -2), Tile(-2, 2), Tile(2, -2), Tile(2, 2)]
```
Comment (JeetKunDoug) [inline model/model.py:167]: I like the way you implemented this! 👍

File: tests/board_tests.py [hunk 2]
```diff
@@ -81,14 +94,86 @@ def test_king_jump_back_right(self):
...
+        self.b.set_piece_at(Tile(5, 5), Piece(Color.RED))
+        print(self.b)
+        self.assertEqual(MoveType.JUMP, self.b.move_piece(Tile(0, 0), Tile(2, 2)))
+        print(self.b)
+        self.assertEqual(MoveType.JUMP, self.b.move_piece(Tile(2, 2), Tile(4, 4)))
+        print(self.b)
+        self.assertEqual(MoveType.JUMP, self.b.move_piece(Tile(4, 4), Tile(6, 6)))
+        print(self.b)
+        self.assertEqual(self.b.turn, Color.RED)
+
```
Comment (JeetKunDoug) [inline tests/board_tests.py:176]: What about a player who _has_ a double-jump but only _wants_ to jump once? Are you ***allowed*** to skip a jump or not?

**Review by JeetKunDoug — COMMENTED** [review]
This looks great - just had one question. Please clarify the rules around jumps, and if this meets them then we can go ahead and I'll approve it.

File: tests/board_tests.py (hunk 2 above)
Comment (adamRoar) [reply tests/board_tests.py:176]: In the rules of checkers, if you have a jump available you ***must*** take it. This will be implemented in the next PR.

Comment (JeetKunDoug) [reply tests/board_tests.py:176]: Ok - can you please update the README and add that to the list, and I'll approve this PR.

**Review by JeetKunDoug — APPROVED** [review]
Looks good - :shipit:

**Pull request merged**